src/Type/PostType/PostTypeType.php

```diff
@@ -42,7 +43,9 @@ public function __construct() {
 		$config = [
 			'name' => self::$type_name,
 			'description' => __( 'An Post Type object', 'wp-graphql' ),
-			'fields' => self::fields(),
+			'fields' => function() {
```

Comment:
@CodeProKid ya, anytime there are "circular" dependencies where a type is referenced inside itself, it needs to be in a closure. Probably best if we update them all for future sake as it's unpredictable where a Type will be referenced in the future.